e2e/homepage.spec.ts

```diff
@@ -7,13 +7,29 @@ test('should display homepage in logged out state', async ({ page }) => {
 
 	await homepageLoggedOut.waitForReady();
 
-	await expect(page).toHaveScreenshot({ fullPage: true });
+	await page.waitForLoadState('networkidle');
+
+	await expect(page).toHaveScreenshot({
+		fullPage: true,
+		animations: 'disabled'
+	});
 });
 
 testWithII('should display homepage in logged in state', async ({ page, iiPage }) => {
 	const homepageLoggedIn = new HomepageLoggedIn({ page, iiPage });
 
 	await homepageLoggedIn.waitForReady();
 
-	await expect(page).toHaveScreenshot({ fullPage: true });
+	await page.waitForLoadState('networkidle');
+
+	const numberOfSlides = await homepageLoggedIn.getNumberOfSlides();
+
+	for (let i = 1; i <= numberOfSlides; i++) {
+		await homepageLoggedIn.navigateToSlide(i);
```

Comment:
I may miss some context, but why do we need to navigate to every slide when we are probably interested in just having the last one displayed? Would a simple `await homepageLoggedIn.navigateToSlide(numberOfSlides);` not work in this case?

Reply:
I thought this would be a better solution. Because basically we not only test if the carousel exists but also if the navigation works. test improvement imo

Reply:
done